package.json

```diff
@@ -8,6 +8,8 @@
     "deploy": "sls deploy",
     "lint": "eslint . --ext ts,js --max-warnings 0",
     "lint:fix": "npm run lint -- --fix",
+    "typecheck": "tsc",
```

Comment:
to run in CI ?

Reply:
Yes. It's a private repo under Notarise org which will run the deployment.